geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ListMembersCommandDUnitTest.java

```diff
@@ -18,86 +18,132 @@
 import static org.apache.geode.management.internal.cli.i18n.CliStrings.LIST_MEMBER;
 import static org.apache.geode.test.junit.rules.GfshCommandRule.PortType.jmxManager;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
 
+import java.util.List;
 import java.util.Properties;
+import java.util.regex.Pattern;
 
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import junitparams.naming.TestCaseName;
 import org.junit.BeforeClass;
 import org.junit.ClassRule;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 
+import org.apache.geode.management.internal.cli.result.CommandResult;
 import org.apache.geode.test.dunit.rules.ClusterStartupRule;
 import org.apache.geode.test.dunit.rules.MemberVM;
 import org.apache.geode.test.junit.rules.GfshCommandRule;
 
-
+@RunWith(JUnitParamsRunner.class)
 public class ListMembersCommandDUnitTest {
-  @ClassRule
-  public static ClusterStartupRule lsRule = new ClusterStartupRule();
-  private static MemberVM locator;
+  private static MemberVM locator1, locator2;
+  private Pattern pattern = Pattern.compile("(.*)locator-0(.*)\\[Coordinator]");
 
   @Rule
-  public GfshCommandRule gfsh = new GfshCommandRule(locator::getJmxPort, jmxManager);
+  public GfshCommandRule gfsh = new GfshCommandRule(locator1::getJmxPort, jmxManager);
+
+  @ClassRule
+  public static ClusterStartupRule lsRule = new ClusterStartupRule();
 
   @BeforeClass
-  public static void setup() throws Exception {
+  public static void setup() {
     Properties properties = new Properties();
     properties.setProperty(GROUPS, "locatorGroup");
-    locator = lsRule.startLocatorVM(0, properties);
-    lsRule.startServerVM(1, "serverGroup1", locator.getPort());
-    lsRule.startServerVM(2, "serverGroup1", locator.getPort());
-    lsRule.startServerVM(3, "serverGroup2", locator.getPort());
+    // First member, becomes the coordinator by default.
+    locator1 = lsRule.startLocatorVM(0, properties);
+    locator2 = lsRule.startLocatorVM(1, properties, locator1.getPort());
+
+    lsRule.startServerVM(2, "serverGroup1", locator1.getPort(), locator2.getPort());
+    lsRule.startServerVM(3, "serverGroup1", locator1.getPort(), locator2.getPort());
+    lsRule.startServerVM(4, "serverGroup2", locator1.getPort(), locator2.getPort());
   }
 
   @Test
-  public void listMembersWithoutConnection() throws Exception {
-    gfsh.disconnect();
+  public void listMembersWithoutConnection() {
+    assertThatCode(() -> gfsh.disconnect()).doesNotThrowAnyException();
```

Comment:
if the code throws any exception, the test would automatically fail. it's probably an overkill to make this assertion.